components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -163,7 +163,10 @@ tweaking their $case/namelist_scream.xml file.
     </homme>
 
     <!-- P3 microphysics -->
-    <p3 inherit="physics_proc_base"/>
+    <p3 inherit="physics_proc_base">
+      <do__prescribed__ccn>true</do__prescribed__ccn>
```

Comment:
@AaronDonahue , you don't need double underscores here. It looks like the real name of this parameter is `do_prescribed_ccn`. The double underscores are for represented spaces (which need to be represented as double underscores because XML does not allow spaces in element tag names), so you'd only need to do it the way you did it if the real name of the parameter was `do prescribed ccn`

Reply:
This should be resolved now, can you check again?